doc/[Old]Metrics.md

```diff
@@ -1,5 +1,5 @@
 
-# **!!! IMPORTANT NOTE !!! The metrics described here are deprecated and will be removed in subsequent releases. They are superseded by [edge metrics V2](BuiltInMetrics.md).**
+# **!!! IMPORTANT NOTE !!! The metrics described here were removed in release 1.1.0. They are superseded by [edge metrics V2](BuiltInMetrics.md).**
```

Comment:
Should we even keep this page? Or just remove it?
@veyalla

Reply:
will follow up with Venkat regarding this file